fix: home page rendering

maaz25925 opened the pull request:
Previous:

![image](https://github.com/DiplomaTechCommunity/Project-Vault/assets/96342418/eefafb74-acb0-40a8-b96c-bd63e26074a7)

Updated:

![image](https://github.com/DiplomaTechCommunity/Project-Vault/assets/96342418/e6335ead-0494-4f34-b4c8-70fa4f97342e)

maaz25925 committed:
refactor: changed according to convention

maaz25925 committed:
fix: "why matter" renders on home page with navbar

maaz25925 committed:
feat: linked website to home page title in navbar

maaz25925 commented:
Kindly review @CODEX108 , @anaskhan28 and @AlfiyaSiddique .
PR made pertaining to issue #18 .

CODEX108 commented:
@maaz25925 , the PR looks fine.
Create an issue for making it responsive and post it on the DTC group so folks interested can contribute, you can ask them to refer the contribution readme.

anaskhan28 commented:
Move all the images in the public folder and then import it in the file before using it in a img tag example: 
``
 import img06 from './06.png
``
``<img src={img06}/>``
This is happening due to the folder structure.

![image](https://github.com/DiplomaTechCommunity/Project-Vault/assets/87796038/3e1c9961-d373-4e9a-9ff5-1477f0ef8906)

maaz25925 committed:
fix: "why matter" page images now rendering
Images were moved from src/assets to public/assets
Changed from <img> to importing the images in WhyMatter.jsx
